src/es/tojimangas/src/eu/kanade/tachiyomi/extension/es/tojimangas/Tojimangas.kt

```diff
@@ -0,0 +1,111 @@
+    override fun popularMangaNextPageSelector(): String = searchMangaNextPageSelector()
+    override fun latestUpdatesNextPageSelector(): String = searchMangaNextPageSelector()
+
+    override fun chapterListSelector(): String = "li .epsleft"
+
+    private fun makeMangaRequest(
+        page: Int,
+        addToBuilder: (HttpUrl.Builder) -> HttpUrl.Builder,
+    ): Request {
+        var url = baseUrl.toHttpUrl().newBuilder().apply {
+            addPathSegment("biblioteca")
+            addPathSegment("page")
+            addPathSegment(page.toString())
+        }
+        return GET(addToBuilder(url).build(), headers)
+    }
+
+    override fun searchMangaRequest(page: Int, query: String, filters: FilterList): Request =
+        makeMangaRequest(page) {
+            it.addQueryParameter("title", query)
+        }
+
+    override fun popularMangaRequest(page: Int): Request = makeMangaRequest(page) {
+        it.addQueryParameter("order", "popular")
+    }
+
+    override fun latestUpdatesRequest(page: Int): Request = makeMangaRequest(page) {
+        it.addQueryParameter("order", "update")
+    }
+
+    override fun searchMangaFromElement(element: Element): SManga = SManga.create().apply {
+        setUrlWithoutDomain(element.attr("href"))
+        title = element.attr("title")
+        thumbnail_url = element.selectFirst("img")?.attr("src")
+    }
+
+    override fun popularMangaFromElement(element: Element): SManga = searchMangaFromElement(element)
+
+    override fun latestUpdatesFromElement(element: Element): SManga =
+        searchMangaFromElement(element)
+
+    private fun getInfoGridValue(infoGridData: String, key: String): String? {
+        return """$key\S+ (\S*)<""".toRegex().find(infoGridData)?.groups?.get(1)?.value
+    }
+
+    override fun mangaDetailsParse(document: Document): SManga = SManga.create().apply {
+        thumbnail_url = document.selectFirst("thumb img")?.attr("src")
+
+        description = document.selectFirst(".desc")
+            ?.select("p:not([style='display: none;'])")
+            ?.filter { p -> p.text() != "" }
+            ?.joinToString { p -> p.text() }
+
+        genre = document.select(".genre-info a").joinToString { a -> a.text() }
+
+        val infoGridData = document.selectFirst(".infox .spe")?.html()
+        if (infoGridData != null) {
+            author = getInfoGridValue(infoGridData, "Autor")
+
+            status = when (getInfoGridValue(infoGridData, "Estado")?.lowercase()) {
+                "activo" -> SManga.ONGOING
+                "finalizado" -> SManga.COMPLETED
+                else -> SManga.UNKNOWN
+            }
+        }
+    }
+
+    private val dateFormat = SimpleDateFormat("dd MMMM, yyyy", Locale("es"))
+
+    override fun chapterFromElement(element: Element): SChapter = SChapter.create().apply {
+        val a = element.selectFirst("a")
+        a?.attr("href")?.let { setUrlWithoutDomain(it) }
+        name = a?.text().toString()
```

Comment:
Make it more explicit that you're treating null as `""`

```suggestion
        name = a?.text() ?: ""
```
```suggestion
        a?.text().also { name = it }
```